allUpperCase.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
```

Comment:
This is where documentation should be :'(

Reply:
done :)